components/ratgdo/switch/ratgdo_switch.cpp

```diff
@@ -32,6 +37,9 @@ namespace ratgdo {
             } else {
                 this->parent_->inactivate_learn();
             }
+        } else if (this->switch_type_ == SwitchType::RATGDO_LED) {
+            this->pin_->digital_write(state);
+            this->publish_state(state);
```

Comment:
I _think_ this is safe here, but need to double check when I'm not in flight

Reply:
[Similar approach as dry contact outputs.](https://github.com/ratgdo/esphome-ratgdo/blob/ec3ba31ce8ebaadefbe6855309ed022d45a80c64/components/ratgdo/dry_contact.cpp#L105)